elfi/methods/post_processing.py

```diff
@@ -0,0 +1,233 @@
+"""
+Post-processing for posterior approximations from other ABC algorithms.
+
+See the review paper:
+
+Fundamentals and Recent Developments in Approximate Bayesian Computation
+Lintusaari et. al
+Syst Biol (2017) 66 (1): e66-e82.
+https://doi.org/10.1093/sysbio/syw077
+
+for more information.
+"""
+
+from sklearn.linear_model import LinearRegression
+import numpy as np
+
+from . import results
+
+
+__all__ = ('LinearAdjustment', 'adjust_posterior')
```

Comment:
Well, I would say this is not proper use of the `__all__` variable then. It specifies the things that will be imported when doing a star import of the module. In this case it is concievable that someone might want to specifically import these to their global namespace. I do not see any benefit to removing `__all__`. `adjust_posterior` is the only thing that is directly under the `elfi` namespace.